compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/SkiaBasedOwner.skiko.kt

```diff
@@ -237,8 +237,26 @@ internal class SkiaBasedOwner(
         }
     }
 
-    var contentSize = IntSize.Zero
-        private set
+    /**
+     * Provides a way to measure Owner's content in given [constraints]
+     * Draw/pointer and other callbacks won't be called here like in [measureAndLayout] functions
+     */
+    internal fun measureInConstraints(constraints: Constraints): IntSize {
+        val oldConstraints = this.constraints
+        try {
+            // TODO: is it possible to measure without reassigning root constraints?
+            measureAndLayoutDelegate.updateRootConstraints(constraints)
+            measureAndLayoutDelegate.measureAndLayout()
```

Comment:
```suggestion
            measureAndLayoutDelegate.measureOnly()
```

We didn't have this function before, but it is more suitable

Reply:
I tried to use this function, but it didn't work for me, since it just reuses previous layout, as I understand. So we anyway need to make a layout

Reply:
It is not what description says. Layout is a about assignment positions to nodes (after we assigned sizes). And positions don't affect `measuredWidth`. What kind of issues you encountered?

I changed this line, and ComposePanelTest still pass